src/exporter.js

```diff
@@ -0,0 +1,125 @@
+import {
+  rand,
+  checksum,
+  getLastItem
+} from './helpers';
+const EXPORTED_METHODS = ['save', 'addMedia', 'addCard', 'update'];
+export const SEPARATOR = '\u001F';
+
+export default class {
+  constructor(db, zip) {
+    this.db = db;
+    this.zip = zip;
+    this.media = [];
+    this.topDeckId = rand();
+    this.topModelId = rand();
+    this.separator = SEPARATOR;
+
+    EXPORTED_METHODS.forEach(methodName => this[methodName] = this[methodName].bind(this));
```

Comment:
Please remove it and `EXPORTED_METHODS`. There is no implication that they should be used without `this` in `master` branch (even though they can be used, it's a hidden from public surface)